Podfile

```diff
@@ -47,10 +47,10 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 4.56.0'
+  # pod 'WordPressKit', '~> 4.56.0'
```

Comment:
Friendly reminder: we'll want to pull in the relevant `WordPressKit-iOS@4.56.1` changes via this line once we merge and publish https://github.com/wordpress-mobile/WordPressKit-iOS/pull/537.